webpack/scenes/RedHatRepositories/components/__tests__/RepositorySetRepositories.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import thunk from 'redux-thunk';
+import { shallow } from 'enzyme';
+import configureMockStore from 'redux-mock-store';
+import RepositorySetRepositories from '../RepositorySetRepositories';
+
+const mockStore = configureMockStore([thunk]);
+const store = mockStore({ katello: { redHatRepositories: { repositorySetRepositories: [] } } });
+
+describe('RepositorySetRepositories Component', () => {
+  it('sorts repos correctly', async () => {
+    const shallowWrapper = shallow(<RepositorySetRepositories
+      store={store}
+      contentId={1}
+      productId={2}
+      type="foo"
+    />);
+
+    const repos = [
+      { arch: 'x86_64', releasever: '5.11' },
+      { arch: 'x86_64', releasever: '7Server' },
+      { arch: 'x86_64', releasever: '7.10' },
+      { arch: 'x86_64', releasever: '7.1' },
+      { arch: 'i386', releasever: '5.11' },
+      { arch: 'i386', releasever: '5Workstation' },
+      { arch: 'x86_64', releasever: '7.11' }];
+
+    const result = shallowWrapper.dive().instance().sortedRepos(repos);
+
+    const expectedIndices = [1, 5, 6, 2, 3, 0, 4];
+
+    expectedIndices.forEach((expected, i) => {
+      expect(result[i]).toEqual(repos[expected]);
+    });
```

Comment:
I know there were no tests before, but we really should have a snapshot comparison for any component. [here](https://git.io/fAQCu) is an example

Reply:
this is a separate test outside of the repo sorting testing, snapshots are a rendered component, which is then used as comparison on any future tests so you can see what has changed in the actual render when you refactor/change anything

Reply:
Something like what I've added in my new commit?